graph/patterns/alternate-key.md

```diff
@@ -2,47 +2,45 @@
 
-`serviceRoot/Users?$filter=email eq 'hello@microsoft.com'`, the returned result is wrapped in an array that needs to be unpacked.
+While it is still possible to use the oData filter, such as
 
+`https://graph.microsoft.com/v1.0/users?filter=mail eq 'bob@contoso.com'`, the returned result is wrapped in an array that needs to be unpacked.
 
 ## Solution
---------
-
-oData offers entity addressing via an alternate key using the same parentheses-style convention as for the canonical key, with one difference: single-part alternate keys MUST specify the key property name to unambiguously determine the alternate key.
-
-http://host/service/Employees(0) - Retrieves the employee with ID = 0
-http://host/service/Employees(email='hello@microsoft.com') Retrieves the employee with the email matching `hello@microsoft.com`
 
-## When to Use this Pattern
-------------------------
+---
 
-This pattern works when the alternate key is good enough to identify a single entity that is properly namespaced; while it does not work if the resultset has more than one element.
+oData offers resource addressing via an alternate key using the same parentheses-style convention as for the canonical key, with one difference: single-part alternate keys MUST specify the key attribute name to unambiguously determine the alternate key.
 
-In such case, we **strongly** advice to throw an exception and encourage the user to use `$filter` rather than returning the first result of the query 
+https://graph.microsoft.com/v1.0/users(0) - Retrieves the employee with ID = 0
+https://graph.microsoft.com/v1.0/users(email='bob@contoso.com') Retrieves the employee with the email matching `bob@contoso.com`
 
-## Example
--------
+## When to Use this Pattern
 
-The same user identified via the alternate key SSN, the canonical (primary) key ID using the non-canonical long form with specified key property name, and the canonical short form without key property name
+---
 
-http://host/service/Employees/1a89ade6-9f59-4fea-a139-23f84e3aef66
+This pattern works and makes sense when the alternate key is good enough to identify a single resource and provides an useful alternative to the client; while it does not work if the resultset has more than one element.
 
-http://host/service/Employees(ssn='123-45-6789')
+In such case, we **strongly** advice to throw an exception and encourage the user to use `$filter` rather than returning the first result of the query
 
-http://host/service/Employees(email='hello@microsoft.com')
+## Example
 
+---
 
+The same user identified via the alternate key SSN, the canonical (primary) key ID using the non-canonical long form with specified key attribute name, and the canonical short form without key attribute name
 
+https://graph.microsoft.com/v1.0/users/1a89ade6-9f59-4fea-a139-23f84e3aef66
 
+https://graph.microsoft.com/v1.0/users(ssn='123-45-6789')
 
+https://graph.microsoft.com/v1.0/users(email='bob@contoso.com')
```

Comment:
Could you provide examples with responses consistent with other patterns like https://github.com/microsoft/api-guidelines/blob/graph/graph/patterns/dictionary.md?